src/rgw/rgw_rados.cc

```diff
@@ -12773,6 +12773,7 @@ int RGWRados::cls_bucket_list(RGWBucketInfo& bucket_info, int shard_id, rgw_obj_
 
   map<string, bufferlist> updates;
   uint32_t count = 0;
+  utime_t now = ceph_clock_now();
```

Comment:
is there a way we can avoid casting to utime_t?

Reply:
well,i will update other expiration prs recently,thx

Reply:
@mattbenjamin  changed it